Anatoly_Katsubo/03.GIT.Hosting/03.GIT.Hosting.md

```diff
@@ -0,0 +1,32 @@
+my repository
```

Comment:
Could you please use git markdown for reports: [https://guides.github.com/features/mastering-markdown/](https://guides.github.com/features/mastering-markdown/)